docker/requirements.txt

```diff
@@ -22,12 +21,12 @@ humanize==0.5.1
 hurry.filesize==0.9
 Jinja2==2.10.1
 kubernetes-asyncio==9.1.0
-kubernetes==9.0.0
 libsass==0.19.2
 nest_asyncio==1.0.0
 parsimonious==0.8.1
 prometheus_async==19.2.0
 prometheus_client==0.7.1
+pyjwt==1.7.1
```

Comment:
Why do you need pyjwt here?

Reply:
Ah, good question.  The relevant code went in in another PR:

https://github.com/hail-is/hail/blob/master/hail/python/hailtop/aiogoogle/auth/credentials.py#L88

To create an authorization token from a service account key, you need to create and sign an "assertion" JWT that says what scopes the token should have access to. 

It is now being deployed for the first time (I ran it locally in cleanup_gcr but I had it installed already)